Modules/cjkcodecs/multibytecodec.h

```diff
@@ -94,6 +94,12 @@ typedef struct {
     _MultibyteStatefulEncoder_HEAD
 } MultibyteIncrementalEncoderObject;
 
+typedef struct {
+    unsigned char pending[MAXENCPENDING];
```

Comment:
Shouldn't this is `MAXPENDING*4`?  4 is max UTF-8 length of Unicode codepoint.

Reply:
Yes, very good point. Thanks.